### DISCUSSION TITLE
Autocommit=False for scoped_session

### DISCUSSION THREAD

**Discussion by reuvenstr** [sqlalchemy/sqlalchemy#12017] [Usage Questions]
Hi,

I'm trying to adjust the codebase to sqlalchemy 2.0, and I see in the documentation that to receive `autocommit=True`, engine `execution_option` should be provided with `isolation_level='autocommit'`.

It helps me in most cases, but there are some places where I want to disable autocommit. How can I achieve it if the only thing that I have are `scoped_session`, `sessionmaker` and `engine`?

Thanks ahead

**Comment by CaselIT** [comment]
Hi,

I believe you can change the autocommit on a per-execution basis, by passing the execution options to a single execute call

**Reply by reuvenstr** [reply]
I will try, thanks
